templates/boilerplates/form.html

```diff
@@ -11,8 +11,12 @@
 
 {% for field in form %}
 <div class="form-group">
-  {{ field.label_tag }}
-
+  {% if field.label_tag == '<label for="id_course">Course:</label>' %}
+    {% comment %} <p>{{ field.value }}</p> {% endcomment %}
+  {% else %}
+    {{ field.label_tag }}
+  {% endif %}
+  
```

Comment:
Refactor this hardcode here.

Reply:
deleted this if condition. hardcode part removed.